src/backend/SyncBackendConnection.cpp

```diff
@@ -743,22 +743,25 @@ bool SyncBackendConnection::update_host(
         models::ListModel* physical_model,
```

Comment:
Line 340: call `get_entities` with the `SyncBackendConnection` method so every `StatisticsBackend` call is correctly encapsulated
Remove try catch